modules/install-sno-monitoring-the-installation-manually.adoc

```diff
@@ -0,0 +1,36 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+[id="monitoring-the-installation-manually_{context}"]
+= Monitoring the installation manually
+
+If you created the ISO manually, use this procedure to monitor the installation.
+
+.Procedure
+
+. Monitor the installation:
++
+[source,terminal]
+----
+$ openshift-install --dir=ocp wait-for install-complete
+----
++
+The server will reboot several times, deploying the control plane.
+
+. After installation completes, check the environment (optional):
```

Comment:
```suggestion
. Optional: After installation completes, check the environment:
```

Reply:
Does the style guide call for this?

Reply:
@johnwilkins Yes. It is in the IBM Style Guide.